docs/src/main/paradox/user/lang/index.md

```diff
@@ -25,16 +25,14 @@ Here is a list of other, **non-official** libraries in various other languages:
 | [scala-support](https://github.com/cloudstateio/cloudstate/tree/master/scala-support/src/main) | Scala | :warning: | :warning: |
 | [kotlin-support](https://github.com/cloudstateio/kotlin-support) | Kotlin | :warning: | :warning: |
 | [cloudstate-csharp](https://github.com/nagytech/cloudstate-csharp)| C# | :heavy_check_mark: | :heavy_check_mark: | 
-| [python-support](https://github.com/marcellanz/cloudstate_python-support/tree/feature/python-support)| Python | | |
-| | Elixir | | |
-| | F# | | |
-| | Ruby | | |
-| | Rust | | |
```

Comment:
Rust already has a repository that you can use if you wish: https://github.com/sleipnir/cloudstate-rust